recipes/visceral-evaluatesegmentation/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: visceral-evaluatesegmentation
+  version: "2015.07.02"
+
+source:
+  fn: EvaluateSegmentation-2015.07.02-Ubuntu.zip # [linux]
+  url: https://github.com/Visceral-Project/EvaluateSegmentation/raw/master/builds/Ubuntu/EvaluateSegmentation-2015.07.02-Ubuntu.zip # [linux]
```

Comment:
Can you compile this one and not using the rebuild-binary? I'm pretty sure this binary will not work on HPC environments.